test/relay_api_test.ts

```diff
@@ -0,0 +1,85 @@
+import * as fetchMock from 'fetch-mock';
+import * as chai from 'chai';
+import * as BigNumber from 'bignumber.js';
+import * as Sinon from 'sinon';
+import {ZeroEx} from '../src/';
+import {chaiSetup} from './utils/chai_setup';
+import {Web3Wrapper} from '../src/web3_wrapper';
+import {SchemaValidator, schemas} from '0x-json-schemas';
+
+chaiSetup.configure();
+chai.config.includeStack = true;
+const expect = chai.expect;
+
+// We need to have a mock API server in order to test that
+describe('Relay API', () => {
+    const relayUrl = 'https://example.com';
+    const relay = new ZeroEx.Relay(relayUrl);
+    const schemaValidator = new SchemaValidator();
+    const responsePayload = {example: 'data'};
```

Comment:
Let's stub out the actual responses we expect, and make sure they are properly converted to usable objects (i.e BigNumbers instead of strings for amounts).